React/wbooks/src/app/screens/Home/components/SearchInput/index.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import searchSvg from '../../assets/search.svg';
+
+import './styles.css';
+
+class SearchInput extends Component {
+  state = {value: ''};
+
+  handleChange = event => {
+    this.setState({ value: event.target.value });
+    this.props.onInputChange(event.target.value);
+  }
+
+  handleFocus = () => {
+    this.setState({ value: '' });
+  }
+
+  handleClick = () => {
+    this.props.onInputChange(this.state.value);
+  }
+
+  render() {
+    const defaultValue = 'Buscar...';
```

Comment:
This const never change. it is not good put inside the render because in each render it is redefined. Move it next to components functions